tests/queries/0_stateless/02354_annoy_index.sql

```diff
@@ -53,30 +55,34 @@ INSERT INTO tab VALUES (1, [0.0, 0.0, 10.0]), (2, [0.0, 0.0, 10.5]), (3, [0.0, 0
 
 -- See (*) why commented out
 -- SELECT 'WHERE type, L2Distance';
--- SELECT *
+-- WITH [0.0, 0.0, 10.0] AS reference_point
```

Comment:
What the point in comment which just duplicates the subsequent query? There is a number of such comments